fe/fe-core/src/main/java/com/starrocks/qe/HDFSBackendSelector.java

```diff
@@ -227,44 +229,66 @@ private long computeTotalSize() {
 
     @Override
     public void computeScanRangeAssignment() throws UserException {
+        computeGeneralAssignment();
+        if (useIncrementalScanRanges) {
```

Comment:
add some commnet for why need add a empty scan range for all woker

Reply:
OK. But i can only give a brief explaination here. detailed explaination is in doc.